### PR TITLE
finished codechallenge 38

### PR DESCRIPTION
**Pull request by tahminaringer**

### CODE REVIEW

**Comment by tahminaringer** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ x] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [ x] Picture of whiteboard

File: javascript/code-challenges/graph/depthFirst/depthFirst.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    for (let edge of neighbors) {
+      if (!visitedNodes.has(edge.verter)) {
+        _traverseNeighbors(edge.vertex);
+      }
+    }
+  };
+  _traverseNeighbors(startNode);
+  return visitedNodes;
+}
+
```
Comment (adrianhuebner) [inline javascript/code-challenges/graph/depthFirst/depthFirst.js:19]: Once you have a chance to get tests written, I will regrade for you!
